README.md

```diff
@@ -87,96 +82,46 @@ Use dash (`-i -`) to read from standard input:
 ```bash
 $ meeshkan build --source file -i - < recordings.jsonl
```

Comment:
I would remove documentation for the `--source`  parameter completely, we can infer the source from the parameter schema (https://github.com/meeshkan/meeshkan/issues/76).